Dockerfile.ubuntu19.04-python3-tensorflow

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:19.04
+
+ENV LANG C.UTF-8
+
+RUN apt-get update \
+    && apt-get install -y build-essential python3-dev python3-pip python3-numpy-dev python3-six \
+       libfftw3-3 libyaml-0-2 libtag1v5 libsamplerate0 python3-yaml \
+       libavcodec58 libavformat58 libavutil56 libavresample4 wget pkg-config libeigen3-dev\
+    && rm -rf /var/lib/apt/lists/*
+
+RUN python3 -m pip install --upgrade pip setuptools \
+    && pip3 install --no-cache-dir tensorflow==1.15.0
+
+RUN apt-get update \
+    && apt-get install -y build-essential git \
```

Comment:
this is missing the installation of all of the -dev packages, which means none of the audioloader algorithms are compiled, and therefore none of the examples either.

When I add these packages, building the examples quits with an error:
```
→ building from /essentia/essentia/src/python
[302/443] Linking build/src/examples/essentia_standard_beatsmarker
13:33:23 runner ['/usr/bin/g++', '-pthread', 'src/examples/standard_beatsmarker.cpp.1.o', '-o/essentia/essentia/build/src/examples/essentia_standard_beatsmarker', '-Wl,-Bstatic', '-Wl,-Bdynamic', '-Lsrc', '-L/usr/local/lib', '-lessentia', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyList_Sort'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyType_IsSubtype'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PySys_WriteStdout'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyObject_HasAttrString'
.... lots more
```
it appears that because we link to pywrap_tensorflow_internal we also need to link against libpython, but this isn't happening.

Reply:
I believe that the correct way to expose `libpython` should be by adding a `Requires: python` line when we [define the pkg-config file from the .so contained in the Tensorflow wheels](
https://github.com/MTG/essentia/blob/0a17738f7f1ac5881a6e80c3095eb7f0f8736252/src/3rdparty/tensorflow/setup_tensorflow.py#L87), but this would require pkg-config to be aware of the `libpython` package in order to `waf configure` sucessfuly, which is the case in my Ubuntu machine but not in the `manylinux` docker images that we use to build the wheels. 

Linking against the Tensorflow libraries contained in the wheels is a hack to make both Essentia and TensorFlow importable from the same Python session but to me, it doesn't make much sense to have the executable examples linked to a Tensorflow library full of Python symbols that are never used and maybe we should not support it.

Why not just dividing the examples and python bindings build into 2 separate steps? 

```
src/3rdparty/tensorflow/setup_from_libtensorflow.sh
waf configure --build-static --with-examples --with-vamp --with-tensorflow
waf install

waf clean

src/3rdparty/tensorflow/setup_from_python.sh
waf configure --build-static --with-python --with-tensorflow
waf install
```

@dbogdanov what is your vision on this?

Reply:
We'll have to download and store tensorflow twice in this case (two versions of the shared library, 800GB), this may be way too much. 

After further discussion with @pabloEntropia, it seems that the best for now would be to patch the tensorflow.pc file in the Docker script after running `src/3rdparty/tensorflow/setup_from_python.sh` adding `Requires: python`.

In the long term, we should also resolve the incompatibility of TensorFlow and Essentia extensions when the latter one is linked to a non-python version of libtensorflow shared library (in this case it's impossible to import both tensorflow and essentia together in Python). We should file an issue on TensorFlow related to that.